### PR TITLE
Use cf option instead of cf.invalidate + cf.distribution_id

### PR DESCRIPTION
**Pull request by bfred-it**
`distribution_id` depends on `invalidate` and `invalidate` depends on `distribution_id`.
In short the property should be one. If set, then I want to invalidate it.

This:

```js
cf: {
  invalidate: true,
  distribution_id: 'qwerty...'
}
```

becomes:

```js
cf: 'qwerty...'
```

### CODE REVIEW

**Comment by bfred-it** [comment]
Not sure why this fails, it works correctly locally 

screen shot 2016-06-09 at 00 04 16

**Comment by jelz** [comment]
@bfred-it quoting docs (https://docs.travis-ci.com/user/environment-variables/):

> Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.

Thank you for your contributions, unfortunately I won't have time before the upcoming weekend to process your code and problems that have arose. Thank you for your patience.

File: README.md [hunk 1]
```diff
@@ -62,12 +62,9 @@ sync({
...
   },
-  // this section is optional:
-  cf: {
-    invalidate: true,
-    // have to be provided if the invalidate key is set to true:
-    distribution_id: 'qwerty...'
-  }
+  // if the distribution id is provided,
+  // its content will be invalidated after the upload
+  cf: 'qwerty...'
```
Comment (jelz) [inline README.md]: can we name this option `cf_distribution_id`?

File: lib/config.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ var DEFAULTS = {
   },
   aws: {concurrency: 10},
   s3: {max_age: 365, s_max_age: 1, remove_remove_surplus: false},
-  cf: {invalidate: false}
+  cf: false
```
Comment (jelz) [inline lib/config.js]: just remove this line

File: test/integration/index.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ var config = {
     s_max_age: 1,
     remove_remote_surplus: true
   },
-  cf: {invalidate: false}
+  cf: false
```
Comment (jelz) [inline test/integration/index.js]: remove this line

File: test/unit/aws.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ test('aws#setConfig', function (t) {
 test('aws#setConfig, w/o CloudFront distribution ID', function (t) {
   prepareStubs();
 
-  aws.setConfig(prepareConfig(working({cf: {invalidate: false}})));
+  aws.setConfig(prepareConfig(working({cf: false})));
```
Comment (jelz) [inline test/unit/aws.js]: can you default to an empty object (`ext = ext || {}`) there: https://github.com/jelz/6xs/blob/master/test/valid_config.js#L15 and then remove the arg?

**Comment by jelz** [comment]
@bfred-it I've added couple of comments to your change.

Please rebase your branch onto `upstream/master`. It shouldn't try to run integration tests for fork PRs any longer. And now there's [a README section about contributing](https://github.com/jelz/6xs#contributing).

File: lib/config.js (hunk 2 above)
Comment (bfred-it) [inline lib/config.js]: When I did this, `config, defaults` would fail on:

```js
  t.equal(prepared.cf_distribution_id, false);
```

**Comment by bfred-it** [comment]
I think I addressed everything. Tests pass, integration still tries and fails

**Comment by bfred-it** [comment]
Ping

**Pull request merged**